modules/util/portal-tools-db-upgrade-client/src/testFunctional/tests/DatabaseUpgradeClient.testcase

```diff
@@ -119,7 +119,7 @@ definition {
 
 		var schemaVersion = ValidateSmokeUpgrade.getSchemaVersion(mysqlStatement = "SELECT schemaVersion FROM Release_ WHERE servletContextName = 'com.liferay.journal.web';");
 
-		if (!(contains(${schemaVersion}, "0.0.1"))) {
+		if (!(contains(${schemaVersion}, "4.1.0"))) {
```

Comment:
@javalosjr96 from which module you got the 4.1.0 version? In this case we are queryin the journal web module, see JournalWebUpgradeStepRegistrator